entries/Globalize.load.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="../entries2html.xsl" ?>
+<entry type="method" name="Globalize.load">
+    <title>Globalize.load()</title>
+    <signature>
+        <argument name="cldrJSONData" type="Object">
+            <desc>A JSON object with CLDR data.</desc>
+        </argument>
+    </signature>
+    <desc>This method allows you to load CLDR JSON locale data.</desc>
+    <longdesc>
+        <p>
+            This method can be called as many time as needed. All passed JSON objects are deeply merged internally.
+        </p>
+    </longdesc>
+    <example>
+        <desc>Load CLDR data.</desc>
+        <code><![CDATA[
+Globalize.load({
+  "main": {
+    "en": {
+      "identity": {
+        "version": {
+          "_cldrVersion": "25",
+          "_number": "$Revision: 91 $"
+        },
+        "generation": {
+          "_date": "$Date: 2014-03-13 22:27:12 -0500 (Thu, 13 Mar 2014) $"
+        },
+        "language": "en"
+      },
+      "dates": {
+        "calendars": {
+          "gregorian": {
+            "months": {
+              "format": {
+                "abbreviated": {
+                  "1": "Jan",
+                  "2": "Feb",
+                  "3": "Mar",
+                  "4": "Apr",
+                  "5": "May",
+                  "6": "Jun",
+                  "7": "Jul",
+                  "8": "Aug",
+                  "9": "Sep",
+                  "10": "Oct",
+                  "11": "Nov",
+                  "12": "Dec"
+                }
+              }
+            },
+            "dayPeriods": {
+              "format": {
+                "wide": {
+                  "am": "AM",
+                  "am-alt-variant": "am",
+                  "noon": "noon",
+                  "pm": "PM",
+                  "pm-alt-variant": "pm"
+                }
+              }
+            },
+            "dateFormats": {
+              "medium": "MMM d, y"
+            },
+            "timeFormats": {
+              "medium": "h:mm:ss a",
+            },
+            "dateTimeFormats": {
+              "medium": "{1}, {0}"
+            }
+          }
+        }
+      },
+      "numbers": {
+        "defaultNumberingSystem": "latn",
+        "symbols-numberSystem-latn": {
+          "group": ","
+        },
+        "decimalFormats-numberSystem-latn": {
+          "standard": "#,##0.###"
+        }
+      }
+    }
+  },
+  "supplemental": {
+    "version": {
+      "_cldrVersion": "25",
+      "_number": "$Revision: 91 $"
+    },
+    "likelySubtags": {
+      "en": "en-Latn-US",
+    }
+  }
+});
+]]></code>
+    </example>
+    <category slug="config"/>
+</entry>
```

Comment:
Nope. It's all XML. As @arthurvr mentions, this is a necessary evil since API docs are highly structured.